app/src/main/java/erevacation/com/userlist/ui/homescreen/list/ListRecyclerAdapter.kt

```diff
@@ -5,12 +5,15 @@ import android.view.LayoutInflater
 import android.view.ViewGroup
 import erevacation.com.userlist.databinding.ViewHolderListBinding
 
-class ListRecyclerAdapter() : RecyclerView.Adapter<ListRecyclerAdapter.ListViewHolder>() {
+class ListRecyclerAdapter(val presenter:ListFragmentPresenter) : RecyclerView.Adapter<ListRecyclerAdapter.ListViewHolder>() {
```

Comment:
This line is to long. Can you move `: RecyclerView.Adapter<ListRecyclerAdapter.ListViewHolder>() {` in second line? The extension sign`:` should  always go with the name of the inherited class.